web/index.html

```diff
@@ -58,5 +78,31 @@
 <body>
     <div style="position: absolute; top: 0; left: 0; right: 0; height: 30px;" id="drag-area"></div>
     <div id="root"></div>
+    <div id="splash" style="position: absolute; top: 0; left: 0; right: 0; background-color: white; width: 100%; height: 100%; display: flex; justify-content: center; align-items: center;">
```

Comment:
Please move these styles to the css file mentioned in a previous comment instead of using inline styles.

Reply:
https://github.com/Expensify/App/pull/12334#discussion_r1011843653